rust/kernel/bindings_helper.h

```diff
@@ -14,6 +16,10 @@
 #include <linux/mm.h>
 #include <uapi/linux/android/binder.h>
 
+#ifdef CONFIG_PROC_FS
+#include "../../fs/proc/internal.h"
```

Comment:
I was using it to get access to the `data` field of `proc_dir_entry` so that we can drop the data when dropping the entry.

Removed this by having the Rust `ProcDirEntry` store it's own copy of the data pointer so that we don't need to extract it from the `proc_dir_entry`.